pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -45,19 +45,27 @@ type BuildRunSpec struct {
 
 // BuildRunStatus defines the observed state of BuildRun
 type BuildRunStatus struct {
-
-	// Conditions
+	// Conditions holds the latest available observations of a resource's current state.
```

Comment:
Do we need to re-generate the apis and crds based on the new comments?

When I re-generated by using the new comment, I can see some `git diff` like:
```
               latestTaskRunRef:
-                description: PodName is the name of the pod responsible for executing
-                  this task's steps.
+                description: "LatestTaskRunRef is the name of the TaskRun responsible
+                  for executing this BuildRun. \n TODO: This should be called something
+                  like \"TaskRunName\""
                 type: string
               reason:
-                description: The Succeeded reason of the TaskRun
+                description: "The Succeeded reason of the TaskRun \n Deprecated: Use
+                  Conditions instead. This will be removed in a future release."
                 type: string
```

Reply:
yep API comments feed into the auto generated open api schema stuff.

that output feeds into what you see when you run `kubectl explain` on any core API type or CRD installed on a given cluster.

Reply:
the `make generate` output ideally is in a separate commit for the PR in question

Reply:
I ran `make generate` (inside GOPATH, see #678) and it didn't produce any new contents, but `make generate-crds` did the trick. Pushed a new commit with the updated generated content. 👍

Reply:
Cool, but the integration test still fail..., pls check.

Reply:
Tests pass now.